scalableminds/webknossos
# Silently ignore duplicate tracing save requests

~~@daniel-wer I wasn’t sure how to test this. You were able to reproduce the error in the first place, could you try if this fixes it? :) Also, my frontend changes might need a cleanup (I just dumped some randomstring generation) Thanks!~~

### URL of deployed dev instance (used for testing):
- https://silentlyignoreduplicatesaverequests.webknossos.xyz

### Steps to test:
- Open tracing and dev console (chrome)
- Set network speed to "Slow 3G"
- Set a node and press "Save"
- Then quickly check the "Offline" box, so that the request will reach the server, but the response won't reach the frontend ("Slow 3G" is helping with that)
- Go online again and press "Save"
- should continue without 409 error

- normal tracing should be unchanged
- tracing in multiple tabs should still give conflict errors and force reload

### Issues:
- fixes #3749 

------
